nomad/assets/dashboards/overview.json

```
@@ -0,0 +1,978 @@
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Total Brokers Ready",
+            "type": "query_value",
+            "width": 18,
+            "x": 75,
+            "y": 57
+        },
+        {
+            "height": 8,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "sum",
+                        "q": "avg:nomad.nomad.broker.total_blocked{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Total Brokers Blocked",
+            "type": "query_value",
+            "width": 18,
+            "x": 75,
+            "y": 67
+        },
+        {
+            "height": 8,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "sum",
+                        "q": "avg:nomad.nomad.broker.total_unacked{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "Total Brokers Unacked",
+            "type": "query_value",
+            "width": 18,
+            "x": 75,
+            "y": 77
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "14",
+            "height": 5,
+            "html": "RPC Metrics",
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 18,
+            "x": 95,
+            "y": 52
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "14",
+            "height": 5,
+            "html": "Consul Metrics",
+            "text_align": "center",
+            "tick": false,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 38,
+            "x": 75,
+            "y": 89
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "14",
+            "height": 5,
+            "html": "Raft Metrics",
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 73,
+            "x": 0,
+            "y": 89
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "14",
+            "height": 5,
+            "html": "Runtime Metrics",
+            "text_align": "center",
+            "tick": false,
```

`Runtime Metrics` pointer is not enabled. Can we also shift the widgets down by 1 pixel in this section to show pointer.
```suggestion
            "tick": true,
```